### PR TITLE
Ant demo page

### PR DESCRIPTION
**Pull request by Arun-42**
fix: #621

### CODE REVIEW

File: packages/react-renderer-demo/src/components/landing-page/landing-page-cards.js [hunk 1]
```diff
@@ -295,6 +295,16 @@ const LandingPageCards = () => {
                 </a>
               </RouterLink>
             </Grid>
+            <Grid item xs={12} md={4} xl={2}>
```
Comment (rvsia) [inline packages/react-renderer-demo/src/components/landing-page/landing-page-cards.js:298]: The previous two icons should be updated and have `md` set to `4` to make a row.

File: packages/react-renderer-demo/src/pages/mappers/ant-component-mapper.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+}
+```
+
+This field will show the error immediately.
+
+## Layout Specification
+
+Ant Design provides horizontal and vertical layouts for form. Layout can be specified by passing a `layout` prop to `formTemplate`. By default it is set to `vertical`.
+
+<!-- ```jsx -->
```
Comment (rvsia) [inline packages/react-renderer-demo/src/pages/mappers/ant-component-mapper.md:43]: Something is missing here. 😄

**Comment by rvsia** [comment]
@Hyperkid123 Hmm, I didn't know about it. :smile:

**Comment by Hyperkid123** [comment]
Yep its not in the dev setup. It is run automatically before docs deployment, but it should be inside the repo (Good practices and all that).

**Comment by Arun-42** [comment]
Sorry, I was away. I haven't run the build command. Should I?

**Comment by Hyperkid123** [comment]
@Arun-42 Yes, please.

**Comment by Arun-42** [comment]
@Hyperkid123 I ran `yarn build` in `react-forms/packages/react-renderer-demo/` ending with error message 
```
✖ 88614 problems (86614 errors, 2000 warnings)
  2 errors and 0 warnings potentially fixable with the `--fix` option.
```
No changes to the tracked files.

**Comment by Hyperkid123** [comment]
@Arun-42 what is the linter error message?
Can you try running just `yarn generate:examples`?

**Comment by Arun-42** [comment]
I think the error message is from `eslint ./src --fix`. I ran just `yarn generate:examples` and it returned the same output. It lists a bunch of linting errors, all in `./src/app/node_modules` I think. It takes a while to run that. Should it be running on `node_modules` though?

**Comment by Hyperkid123** [comment]
Oh yeah it seems its not excluded for some reason. I will check thy is that.

**Comment by Hyperkid123** [comment]
@Arun-42 I've just tried it locally and both build and generate:examples scripts work on master. I will try it on this PR.

**Comment by Hyperkid123** [comment]
@Arun-42 I've just tried these commands on this branch and it works fine. In what directory you run them?

Make sure you are in `packages/react-renderer-demo/`

Anyway, it did not net greater any new content. Seems you are have written all manually so we can merge.

**Comment by rvsia** [comment]
Thank you for this PR, @Arun-42! :tada:

**Comment by Arun-42** [comment]
Hmm, I did run it in `packages/react-renderer-demo`. Not sure what I did wrong :thinking: .

**Comment by Hyperkid123** [comment]
We will never know :sparkler:

**Pull request merged**